src/client/driver/index.js

```diff
@@ -10,6 +10,7 @@ import * as browser from '../browser';
 import executeActionCommand from './command-executors/execute-action-command';
 import executeWaitForElementCommand from './command-executors/execute-wait-for-element-command';
 import executeNavigateToCommand from './command-executors/execute-navigate-to-command';
+import executePendingBrowserWindowCommand from './command-executors/execute-pending-browser-window-command';
```

Comment:
Well, feel free to name it whatever you want. The main requirement: we should understand after e.g. 3 months how's this thing works and able to re use it's machinery.